fastembed/embedding.py

```diff
@@ -95,13 +95,26 @@ def __init__(
         model_path = self.path / "model.onnx"
         optimized_model_path = self.path / "model_optimized.onnx"
 
+        xenova_model_path = self.path / "onnx" / "model.onnx"
+        xenova_optimized_model_path = self.path / "onnx" / "model_optimized.onnx"
```

Comment:
Let's not use xenova as the prefix, and something else e.g. transformerjs_model_path?

@Xenova primarily works on that project?

Reply:
How about we do https://github.com/qdrant/fastembed/pull/102/commits/8fd3dd31b2a6d467680811bc4871bdf060f216e8 instead?